### PR TITLE
Fix broken warning on package layout page

### PR DESCRIPTION
**Pull request by timsneath**
Fix broken warning at the end of the document

### CODE REVIEW

**Review by jonasfj — APPROVED** [review]

**Comment by timsneath** [comment]
@sfshaza2 I don't think this change should have caused a staging failure, but perhaps you have a magic insight?

File: src/tools/pub/package-layout.md [hunk 1]
```diff
@@ -522,11 +522,13 @@ This helps speed-up future re-runs of the build steps.
 {{site.alert.warning}}
   When developing a tool that wants to cache files in `.dart_tool/`,
   ensure the following:
+  
     * You are using a subdirectory named after a package you own
        (`.dart_tool/<my_tool_package_name>/`)
     * Your files don't belong under source control, 
        as `.dart_tools` is generally listed in `.gitignore`
-{{site.alert.warning}}
+
```
Comment (parlough) [inline src/tools/pub/package-layout.md:530]:
```suggestion
```

**Review by parlough — APPROVED** [review]
Thanks for catching and fixing this!

**Comment by parlough** [comment]
The staging failure is due to the Dart 2.18 builds being released. I'll fix that in a separate PR :)

**Pull request merged**